LeanplumTests/LPGetVarsApiTest.m

```diff
@@ -0,0 +1,164 @@
+//
+//  LPGetVarsApiTest.m
+//  LeanplumTests
+//
+//  Created by Mayank Sanganeria on 5/22/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <OHHTTPStubs/OHHTTPStubs.h>
+#import <OHHTTPStubs/OHPathHelpers.h>
+#import "LPGetVarsApi.h"
+#import "LPAPIConfig.h"
+#import "LPConstants.h"
+#import "LPTestHelper.h"
+
+@interface LPGetVarsApiTest : XCTestCase
+
+@end
+
+@implementation LPGetVarsApiTest
+
+- (void)setUp {
+    [super setUp];
+    [LPTestHelper setup];
+}
+
+- (void)tearDown {
+    [super tearDown];
+    [OHHTTPStubs removeAllStubs];
+}
+
+- (void)testGetVarsApi {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Query timed out."];
+    [LPGetVarsApi getVarsWithParameters:nil success:^ {
+        [expectation fulfill];
+    } failure:^(NSError *error) {
+    }];
+    
+    [self waitForExpectationsWithTimeout:30.0 handler:^(NSError *error) {
+        if (error) {
+            NSLog(@"Error: %@", error);
+        }
+    }];
+}
+
+- (void)testGetVarsApiWithParameters {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Query timed out."];
+    NSDictionary *params = @{ @"testKey": @"testValue" };
+    [LPGetVarsApi getVarsWithParameters:params success:^ {
+        [expectation fulfill];
+    } failure:^(NSError *error) {
+    }];
+    
+    [self waitForExpectationsWithTimeout:10.0 handler:^(NSError *error) {
+        if (error) {
+            NSLog(@"Error: %@", error);
+        }
+    }];
+}
+
+//TODO : This is passing
```

Comment:
?

Reply:
so basically, there is no HTTP error, even though the dev key is empty